server/datastore/mysql/migrations_test.go

```diff
@@ -34,3 +38,95 @@ func TestMigrationStatus(t *testing.T) {
 	require.Nil(t, err)
 	assert.EqualValues(t, fleet.AllMigrationsCompleted, status)
 }
+
+func TestMigrations(t *testing.T) {
+	// Create the database (must use raw MySQL client to do this)
+	ds := createMySQLDSForMigrationTests(t, t.Name())
+	defer ds.Close()
+
+	require.NoError(t, ds.MigrateTables())
+
+	// Dump schema to dumpfile
+	cmd := exec.Command(
+		"docker-compose", "exec", "-T", "mysql_test",
+		// Command run inside container
+		"mysqldump", "-u"+testUsername, "-p"+testPassword, "TestMigrations", "--compact", "--skip-comments",
+	)
+	var stdoutBuf bytes.Buffer
+	cmd.Stdout = &stdoutBuf
+	require.NoError(t, cmd.Run())
+
+	require.NotEmpty(t, stdoutBuf.String())
+}
+
+func createMySQLDSForMigrationTests(t *testing.T, dbName string) *Datastore {
```

Comment:
We need migration tests "contained", so I moved them here. There was an issue with `ds.Drop` in the `TestMigrationStatus`, so migrations were failing.